src/redux/toDoSaga.js

```diff
@@ -0,0 +1,51 @@
+import { call, put, takeEvery } from "redux-saga/effects";
+import axios from "axios";
+import { loadToDos } from "./toDoSlice/toDoSlice";
+
+function* addToDoJSON(action) {
+  const { payload } = action;
+  yield call(() => axios.post("http://localhost:8000/toDos", payload));
```

Comment:
This seems duplication http://localhost:8000.
make a constant variable 
basePath= http://localhost:8000 and use that variable in all your api calls